tests/email.test.js

```diff
@@ -0,0 +1,46 @@
+const expect = require('unexpected');
+const MaskData = require('../index');
+
+describe('Masking email', function() {
+  describe('with default options', function() {
+
+    const defaultEmailMaskOptions = {
+      maskWith: "*",
+      unmaskedStartCharacters: 3,
+      unmaskedEndCharacters: 1,
+      maskAtTheRate: false,
+      maxMaskedCharactersBeforeAtTheRate: 10,
+      maxMaskedCharactersAfterAtTheRate: 10,
+    };
+
+    const tests = [
+      {
+          input:  'my.testEmail@testMail.com', 
+          output: 'my.*********@***********m'
+      },
+      {
+          input:  'my@test.com', 
+          output: 'my@*******m'
+      },
+      {
+          input:  'a@b.cd', 
+          output: '*@*.**'
+      }
+    ];
+
+    tests.forEach(function(test) {
+      // TODO: If they are defaults, why do they need to be arguments?
```

Comment:
For masking with default options, you can use MaskData.simpleEmailMask(email) 

Thanks for the suggestions. It is confusing to have multiple mask methods. so I will work on optional mask configuration for the same method.  If the configuration is not provided, then it will take default value itself. And I will depricate simpleMask() methods.